packages/common/src/utils/faker/fakeCertificateData.ts

```diff
@@ -280,6 +281,27 @@ export const fakeTextAreaElement = (
     children
   )
 
+export const fakeTextAreaAccordionElement = (
+  data?: PartialCertificateDataElement<ConfigUeTextArea, ValueText>,
+  children?: CertificateData[]
+): CertificateData =>
+  fakeDataElement(
+    {
+      ...data,
+      config: {
+        type: ConfigTypes.UE_TEXTAREA,
+        accordion: {
+          openText: 'Visa fritextfält',
+          closeText: 'Dölj fritextfält',
+          header: faker.lorem.sentence(5),
+        },
+        ...data?.config,
+      },
+      value: { type: CertificateDataValueType.TEXT, text: 'Text', limit: 50, ...data?.value },
+    },
+    children
+  )
```

Comment:
We should not create fakeElement function for individual configuration parameter. This should be removed and used with fakeTextAreaElement instead. 

```typescript
fakeTextAreaElement({
  config: {
    accordion: {  
      openText: 'Visa fritextfält',
      closeText: 'Dölj fritextfält',
      header: faker.lorem.sentence(5),
    }
  }
})
```